mne/io/fil/tests/test_fil.py

```diff
@@ -140,3 +143,21 @@ def test_fil_all():
     _fil_megmag(raw, mat)
     _fil_stim(raw, mat)
     _fil_sensorpos(raw, mat)
+
+
+@testing.requires_testing_data
+def test_fil_no_positions(tmp_path):
+    """Test FIL reader in cases where a position file is missing."""
+    test_path = tmp_path / "FIL"
+    shutil.copytree(fil_path, test_path)
+
+    posname = test_path / "sub-noise_ses-001_task-noise220622_run-001_positions.tsv"
+    binname = test_path / "sub-noise_ses-001_task-noise220622_run-001_meg.bin"
+
+    remove(posname)
+
+    with pytest.warns(RuntimeWarning, match="No sensor positions*"):
```

Comment:
This will match one or more s's at the end, I think you meant:
```suggestion
    with pytest.warns(RuntimeWarning, match="No sensor positions.*"):
```
It won't matter here but might as well keep it cleaner